vertical-pod-autoscaler/hack/generate-crd-yaml.sh

```diff
@@ -32,7 +32,7 @@ trap cleanup EXIT
 if [[ -z $(which controller-gen) ]]; then
     (
         cd $WORKSPACE
-	      go install sigs.k8s.io/controller-tools/cmd/controller-gen@0.9.2
+	      go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2
```

Comment:
This is weird that we have to change this... Does that mean that upstream master is not working for that part?

Reply:
I think they haven't updated the CRDs in a long time ? Clearly the `0.9.2` tag doesn't not exist